apps/core-app/src/main/modules/box-tool/addon/files/file-provider.ts

```
@@ -3362,6 +3362,14 @@ class FileProvider implements ISearchProvider<ProviderContext> {
     )
   }
 
+  public isSearchIndexWorkerBusy(): boolean {
+    return (
+      this.pendingIndexWorkerResults.size > 0 ||
```

ESLint

[error] '||' should be placed at the beginning of the line.

(style/operator-linebreak)